src/did.ts

```diff
@@ -0,0 +1,176 @@
+import * as base58 from 'base58-universal/main.js'
+import { CryptoSystem, Msg } from 'keystore-idb/types'
+
+import eccOperations from 'keystore-idb/ecc/operations'
+import rsaOperations from 'keystore-idb/rsa/operations'
+import utils from 'keystore-idb/utils'
+
+import * as dns from './dns'
+import * as keystore from './keystore'
+import { arrbufs } from './common'
+
+
+const ECC_DID_PREFIX: ArrayBuffer = new Uint8Array([ 0xed, 0x01 ]).buffer
+const RSA_DID_PREFIX: ArrayBuffer = new Uint8Array([ 0x00, 0xf5, 0x02 ]).buffer
+const BASE58_DID_PREFIX: string = 'did:key:z'
+
+
+
+// KINDS
+
+
+/**
+ * Create a DID to authenticate with.
+ */
+export async function local(): Promise<string> {
+  const ks = await keystore.get()
```

Comment:
🤔 Why is `keystore.get()` async? Does it set one up if it doesn't exist yet?

Reply:
Indeed.